src/styles/main.scss

```diff
@@ -1,3 +1,54 @@
+@import "./variables";
+
 body {
   margin: 0;
+  padding-top: 50px;
+}
+
+.calendar {
+  margin: auto;
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  width: $calendar-width;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    height: $cell-size;
+    width: $cell-size;
+    background-color: #eee;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)